tests/modules/nf-core/samtools/merge/test.yml

```diff
@@ -5,6 +5,7 @@
     - samtools
   files:
     - path: output/samtools/test.bam
+    - path: output/samtools/test.bam.csi
```

Comment:
shouldn't the index be bai with small files?

Reply:
By default SAM and BAM will use CSI indices.

Reply:
Emphasis mine:

> The --write-index option enables automatic index creation while writing out BAM, CRAM or bgzf SAM files. Note to get compressed SAM as the output format you need to manually request a compression level, otherwise all SAM files are uncompressed. *By default SAM and BAM will use CSI indices while CRAM will use CRAI indices.* If you need to create BAI indices note that it is possible to specify the name of the index being written to, and hence the format, by using the filename##idx##indexname notation.

Reply:
ok, so could still have bai then
Can that be added in this PR?

Reply:
Yes, it seems possible. but for a test, it will be very ugly.

Reply:
ok, then let's not worry about it for now, I'll create an issue and add it myself

Reply:
Thanks. Better for another PR, can we merge this one please?